### PR TITLE
agent: add test coverage for find_process function

### PR DESCRIPTION
**Pull request by Shensd**
Add test coverage for the function find_process in src/sandbox.rs.

Signed-off-by: Jack Hance <jack.hance@ndsu.edu>

### CODE REVIEW

File: src/agent/src/sandbox.rs [hunk 1]
```diff
@@ -851,4 +851,59 @@ mod tests {
         let p = s.find_container_process("not-exist-cid", "");
         assert!(p.is_err(), "Expecting Error, Got {:?}", p);
     }
+
+    #[tokio::test]
+    async fn find_process_exists() {
```
Comment (liubin) [inline src/agent/src/sandbox.rs:856]: Normally test function starts with `test_`.

File: src/agent/src/sandbox.rs [hunk 2]
```diff
@@ -851,4 +851,59 @@ mod tests {
...
+
+        // then test if it founds the correct process
+        assert_eq!(
+            found_process.pid, test_pid,
+            "Should be able to find correct process"
+        );
+    }
+
+    #[tokio::test]
+    async fn find_process_doesnt_exist() {
```
Comment (liubin) [inline src/agent/src/sandbox.rs:891]: These two cases is not very big, so they can be put in one function.

**Comment by arronwy** [comment]
Can one of the admins verify this patch?

**Comment by Shensd** [comment]
I made the edits recommended by @liubin

**Comment by Shensd** [comment]
Changes are now being discussed in #4137.

**Pull request closed without merging**